src/Creature.js

```diff
@@ -107,10 +110,18 @@ export default class Creature {
 
     const nearestFood = getNearestDetails(this, this.state.food, true);
 
-    if (nearestCreature && nearestCreature.ref.canEat(this) && nearestCreature.distance < this.size * 2) {
+    if ((nearestCreature && nearestCreature.ref.canEat(this) && nearestCreature.distance < this.size * 2) || this.scared) {
```

Comment:
```suggestion
    if (this.scareTime !== null || (nearestCreature && nearestCreature.ref.canEat(this) && nearestCreature.distance < this.size * 2)) {
```
This is what I mean by checking for null to see if they are scared. this entire if statement needs refactoring but thats another issue haha.